lib/Command/InstallCommand.php

```diff
@@ -35,6 +35,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 $this->installer->requireExtensions((array) $input->getArgument('extension'));
             } catch (CouldNotInstallExtension $couldNotInstall) {
                 $output->writeln('<error>Could not install</>');
+
+                if ($output->getVerbosity() >= OutputInterface::VERBOSITY_VERBOSE) {
+                    throw $couldNotInstall;
```

Comment:
we should probably include the error message after `Could not install` (not sure why that wasn't there in the first place)

Reply:
well, even on a degree of "normal" verbosity, we will have the message anyway. Check the screens on the first post

Reply:
yeah, I guess that's the reason I didn't show the error message in the first place, so this is good.